aggregator/src/util.rs

```diff
@@ -9,29 +13,28 @@ use snark_verifier::loader::halo2::halo2_ecc::halo2_base::{
     gates::{flex_gate::FlexGateConfig, GateInstructions},
     AssignedValue, Context,
 };
-
-use crate::{
-    aggregation::RlcConfig,
-    constants::{DIGEST_LEN, INPUT_LEN_PER_ROUND, MAX_AGG_SNARKS},
-    DEFAULT_KECCAK_ROWS, NUM_ROUNDS,
+use zkevm_circuits::keccak_circuit::keccak_packed_multi::{
+    get_num_rows_per_round, get_num_rows_per_update,
 };
 
-use std::env::var;
+// Calculates the maximum keccak updates (1 absorb, or 1 f-box invoke)
+// needed for the number of snarks
+pub(crate) fn get_max_keccak_updates(max_snarks: usize) -> usize {
```

Comment:
max_snarks is a constant `MAX_AGG_SNARKS`. should it be a parameter of the function?

Reply:
MAX_AGG_SNARKS is deeply embedded as part of circuit. Not east to change. This time it can be kept as a CONSTANT

Reply:
seems ok to me.  get_max_keccak_updates is a helper function to calculate some number with that input.

Reply:
Advised as ok.